### ISSUE TITLE
Flag to force named lights to have their bounce lighting baked with the rest of the lightmap pages

### ISSUE THREAD

**Issue by Xenthio** [Facepunch/garrysmod-requests#1985]
### Details
Too lazy to type it up so I'll post this screencap.
![image](https://user-images.githubusercontent.com/28588188/176118319-35af50c0-d817-4648-8ba8-af05a28e9c09.png)

**Comment by Jaffies** [comment]
That'll be so much good for the game so rubat won't do it

**Comment by shadow7483147** [comment]
Yeah, if compile time is the main affliction it'd be nice. Either you get crappy pattern lighting and maybe 2 toggleable lights, or good pattern lighting and 1 toggleable light in an area.

**Comment by Xenthio** [comment]
> Yeah, if compile time is the main affliction it'd be nice. Either you get crappy pattern lighting and maybe 2 toggleable lights, or good pattern lighting and 1 toggleable light in an area. Since the only reason Valve decided to disable bounce light for named lights was because it'd generate huge pages exponentially.

i kinda meant that the bounced light is just left there when a light is turned off, kinda like how you can do lights in source 2, it looks decent in some situations anyways.

**Comment by JarosLucky** [comment]
@Xenthio extend this issue with content from #2663

**Comment by Xenthio** [comment]
I have modded the mp branch version of vrad to be able to do this, but unfortunately I cannot do this for the version included in gmod, which has many many improvments and also has the source unavailable 64 bit compiler.

the diff for vrad to do the functionality I would like is provided here, though it doesn't lock it behind a launch argument. But i can't imagine that would be difficult to add on.

![Image](https://github.com/user-attachments/assets/13f78a46-0d6c-4564-9dc6-e9c50c901a08)
![Image](https://github.com/user-attachments/assets/e538c7b9-ad84-400c-8813-89cc9fbc8a56)

[alwaysbakeindirectlight.diff.txt](https://github.com/user-attachments/files/19198616/alwaysbakeindirectlight.diff.txt)

**Comment by Xenthio** [comment]
> [@Xenthio](https://github.com/Xenthio) extend this issue with content from [#2663](https://github.com/Facepunch/garrysmod-requests/issues/2663)

I feel extremely silly at duplicating my OWN ISSUE.

**Comment by JarosLucky** [comment]
Probably after tf2 sdk realised and source sdk gwt update, we get 64x tools? You can try PR this to Source SDK or just have it as our fork

**Comment by robotboy655** [comment]
Added -bouncefromnamed to VRAD
